pyaerocom/io/read_mscw_ctm.py

```diff
@@ -1,7 +1,11 @@
+from posixpath import join
```

Comment:
I do not see this import being used

Reply:
That is an old import I ended up not using. I'm removing it now